SPECS/rpm/rpm.spec

```diff
@@ -107,31 +110,38 @@ Python3 rpm.
 %autosetup -n rpm-%{name}-%{version}-release -p1
 
 %build
-sed -i '/define _GNU_SOURCE/a #include "../config.h"' tools/sepdebugcrcfix.c
 # pass -L opts to gcc as well to prioritize it over standard libs
 sed -i 's/-Wl,-L//g' python/setup.py.in
 sed -i '/library_dirs/d' python/setup.py.in
 sed -i 's/extra_link_args/library_dirs/g' python/setup.py.in
 
 ./autogen.sh --noconfigure
+
 %configure \
-    CPPFLAGS='-I/usr/include/nspr -I/usr/include/nss -DLUA_COMPAT_APIINTCASTS' \
-        --program-prefix= \
-        --disable-dependency-tracking \
-        --disable-static \
-        --with-vendor=mariner \
-        --enable-python \
-        --with-cap \
-        --with-lua \
-        --disable-silent-rules \
-        --with-external-db \
-        --with-selinux
+    CPPFLAGS='-DLUA_COMPAT_APIINTCASTS' \
+    --program-prefix= \
+    --with-crypto=openssl \
+    --enable-ndb \
+    --disable-dependency-tracking \
+    --disable-static \
+    --with-vendor=mariner \
+    --enable-python \
+    --with-cap \
+    --disable-silent-rules \
+    --with-selinux
+
+# Remove manpages translations
+rm -r docs/man/{fr,ja,ko,pl,ru,sk}
+
 make %{?_smp_mflags}
```

Comment:
Nit: Line can be replaced with `%make_build` to get standard SMP flags and verbosity boilerplate options